PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'account.fiscal.position'
-    _inherit = 'res.company'
-    _inherit = 'res.config.settings'
-    _inherit = 'res.partner'

Total Changes: 8

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/base_vat/models/account_fiscal_position.py
+++ b/addons/base_vat/models/account_fiscal_position.py
@@ -5,7 +5,7 @@ from odoo.exceptions import ValidationError
 
 
 class AccountFiscalPosition(models.Model):
-    _inherit = 'account.fiscal.position'
+    _inherit = ['account.fiscal.position']
 
     @api.model_create_multi
     def create(self, vals_list):

--- a/addons/base_vat/models/res_company.py
+++ b/addons/base_vat/models/res_company.py
@@ -5,6 +5,6 @@ from odoo import fields, models
 
 
 class ResCompany(models.Model):
-    _inherit = 'res.company'
+    _inherit = ['res.company']
 
     vat_check_vies = fields.Boolean(string='Verify VAT Numbers')

--- a/addons/base_vat/models/res_config_settings.py
+++ b/addons/base_vat/models/res_config_settings.py
@@ -4,7 +4,7 @@ from odoo import fields, models
 
 
 class ResConfigSettings(models.TransientModel):
-    _inherit = 'res.config.settings'
+    _inherit = ['res.config.settings']
 
     vat_check_vies = fields.Boolean(related='company_id.vat_check_vies', readonly=False,
         string='Verify VAT Numbers')

--- a/addons/base_vat/models/res_partner.py
+++ b/addons/base_vat/models/res_partner.py
@@ -87,7 +87,7 @@ _region_specific_vat_codes = {
 
 
 class ResPartner(models.Model):
-    _inherit = 'res.partner'
+    _inherit = ['res.partner']
 
     vies_valid = fields.Boolean(
         string="Intra-Community Valid",
